profiles/base.yaml

```diff
@@ -1403,3 +1403,20 @@ burgle_settings:
 # Attempts to use ;burgle during Athletics, Locksmithing, Thievery, or Stealth
 # Uses burgle_settings for entry method
 train_with_burgle: false
+
+# To group all Duskruin related settings.
+# https://elanthipedia.play.net/Drathrok%27s_Duskruin
+duskruin:
```

Comment:
This also needs an entry in `base-empty.yaml`

Reply:
Sorry, I misunderstood how to populate that file. I get it now :)

Should be fixed per https://github.com/rpherbig/dr-scripts/pull/4390/commits/dbd157fcbc0fef8a53c5460a0c6fe0815ef9200d